### PR TITLE
add specificity to bootstrap yaml execution

### PR DESCRIPTION
**Pull request by reubenson**
bugfix for component bootstrap failing on startup due to my local username (reuben.son) throwing off the parser for yaml directory

### CODE REVIEW

File: lib/bootstrap.js [hunk 1]
```diff
@@ -30,10 +30,8 @@ var log = require('./services/logger').setup({
 function getConfig(dir) {
   dir = path.resolve(dir);
 
-  if (dir.indexOf('.', 2) > 0) {
-    // remove extension
-    dir = dir.substring(0, dir.indexOf('.', 2));
-  }
+  // remove extension
+  dir = dir.replace(/.(yaml|yml)/, '');
```
Comment (jpope19) [inline lib/bootstrap.js]: Should we add a case-insensitive flag?

Comment (reubenson) [reply lib/bootstrap.js]: oh yea, i guess why not!

Comment (reubenson) [inline lib/bootstrap.js]:
```suggestion
  dir = dir.replace(/.(yaml|yml)i/, '');
```

File: lib/bootstrap.js [hunk 2]
```diff
@@ -30,16 +30,16 @@ var log = require('./services/logger').setup({
...
-  }
+  // remove extension
+  dir = dir.replace(/.(yaml|yml)/i, '');
 
   if (files.isDirectory(dir)) {
     // default to bootstrap.yaml or bootstrap.yml
     dir = path.join(dir, 'bootstrap');
   }
 
+  console.log('dir', dir);
```
Comment (jpope19) [inline lib/bootstrap.js]: remove log 🙈

**Pull request merged**